tdrs-backend/gunicorn_start.sh

```diff
@@ -13,8 +13,8 @@ else
 fi
 
 #
-echo "Applying database migrations"
-python manage.py migrate
+#echo "Applying database migrations"
+#python manage.py migrate
 #python manage.py populate_stts
 #python manage.py collectstatic --noinput
 
```

Comment:
I would say go ahead and remove all these commented out lines. If this works as expected, we won't need this reminder. If it doesn't, we will not merge, or will revert.